pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -1128,6 +1131,12 @@ public PublishRateLimiter getBrokerPublishRateLimiter() {
         return brokerService.getBrokerPublishRateLimiter();
     }
 
+    /**
+     * @deprecated Avoid using the deprecated method
+     * #{@link org.apache.pulsar.broker.resources.NamespaceResources#getPoliciesIfCached(NamespaceName)} and we can use
+     * #{@link AbstractTopic#updateResourceGroupLimiter(Policies)} to instead of it.
+     */
+    @Deprecated
```

Comment:
Why not just fix this method?

Reply:
I am afraid of some protocol handler will use this API.

Reply:
Do you mean they want to keep the same behavior(get from cache), and don't want this fix? We can avoid changing the method signature, right?

Reply:
All of the pulsar logic don't use this method anymore. I just want to leave it for some compatible concerns.